index.html

```diff
@@ -30,7 +30,7 @@ <h1>Communities</h1>
             <p>These communities are hosted on Codidact currently:</p>
             <div class="grid community-list">
                 <div class="grid--cell is-4 is-6-md is-12-sm">
-                    <div class="widget">
+                    <div class="widget" name="writing">
```

Comment:
Let's make these attributes `data-name` rather than plain `name` for technical compliance with the HTML spec. Once that's done, I'm happy with this.